package.json

```diff
@@ -72,8 +72,8 @@
     "typescript": "5.0.4"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.0.0",
-    "@nestjs/core": "^9.0.0"
+    "@nestjs/common": "^9.0.0 || ^10.0.0",
```

Comment:
Line 46,47 also need a change.

Reply:
The CI will be running tests with 9.0, not sure if that would break something, so I would at least need to test it with 10 to make sure it still works.

Reply:
Updated all devDependencies related to nestjs with d27a3e09b268442c0c444f9814fd8e791aa26d19